Plugin~/Src/MeshSync/msServer.cpp

```diff
@@ -155,6 +155,10 @@ int Server::processMessages(const MessageHandler& handler)
             m_current_live_edit_request = req;
             handler(Message::Type::RequestServerLiveEdit, *mes);
         }
+        else if (auto command = std::dynamic_pointer_cast<EditorCommandMessage>(mes)) {
+            m_current_command = command;
```

Comment:
won't there be a race condition when setting the `reply` variable if, for example: 
1. there are two incoming EditorCommandMessages
2. the processing of the first message is somehow delayed (300ms timeout)

Reply:
Good call, will implement a dictionary to manage multiple requests

Reply:
Done